Allow using cross-compilers to bootstrap GHC.

In my case I'm using "cross"-compiler from
x86_64-unknown-linux to x86_64-HEAD-linux.

Those targets have the same ABI and can boot one another.
diff --git a/configure.ac b/configure.ac
index a8b8681a36..791367e8bf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -228,11 +228,11 @@ FPTOOLS_SET_PLATFORM_VARS
 if test "$BuildPlatform" != "$bootstrap_target"
 then
     echo "This GHC (${WithGhc}) does not generate code for the build platform"
     echo "   GHC target platform    : $bootstrap_target"
     echo "   Desired build platform : $BuildPlatform"
-    exit 1
+    #exit 1
 fi
 
 # Testing if we shall enable shared libs support on Solaris.
 # Anything older than SunOS 5.11 aka Solaris 11 (Express) is broken.
 
